db/system_keyspace.cc

```diff
@@ -2636,22 +2637,14 @@ future<service::topology> system_keyspace::load_topology_state() {
             // Sanity check for CDC generation data consistency.
             {
                 auto gen_rows = co_await execute_cql(
-                    format("SELECT count(range_end) as cnt, num_ranges FROM system.{} WHERE id = ?",
-                           CDC_GENERATIONS_V3),
+                    format("SELECT count(range_end) FROM {}.{} WHERE key = '{}' AND id = ?",
+                           NAME, CDC_GENERATIONS_V3, CDC_GENERATIONS_V3_KEY),
                     gen_uuid);
```

Comment:
since we can no longer do the count sanity check, let's at least `debug` print the gen uuid and the count.

Reply:
I added that.